deploy/awstools/awstools.py

```diff
@@ -209,6 +209,56 @@ def aws_resource_names() -> Dict[str, Any]:
     return base_dict
 
 
+def farm_security_group_setup() -> None:
+    """Create the security group for build/run farm instances, if it doesn't
+    already exist."""
+
+    aws_resource_names_dict = aws_resource_names()
+    securitygroupname = aws_resource_names_dict['securitygroupname']
+    vpcname = aws_resource_names_dict['vpcname']
+
+    ec2 = boto3.resource('ec2')
+    client = boto3.client('ec2')
+
+    operation_params = {
+        'Filters': [{'Name':'group-name', 'Values': [securitygroupname]}]
+    }
+    firesimsecuritygroup = depaginated_boto_query(client, 'describe_security_groups', operation_params, 'SecurityGroups')
+
+    if len(firesimsecuritygroup) > 1:
+        rootLogger.critical(f"Too many security groups named {securitygroupname}. Exiting.")
+    elif len(firesimsecuritygroup) == 1:
```

Comment:
Can you add a print here indicating that the security group was found and thus not created.

Reply:
Done